### PR TITLE
chore(content-releases): remove future flags

### PR DESCRIPTION
**Pull request by markkaylor**
### What does it do?

- Removes the future flag from content-releases
- Optionally chains .options.draftAndPublish

### Why is it needed?

- The feature is stable

### How to test it?

- Remove the features config for future.contentReleases
- Use a license that has the feature 
- Releases should function as normal

### CODE REVIEW

**Review by simotae14 — APPROVED** [review]
It works

**Comment by markkaylor** [comment]
@Feranchz could you please review and QA:

- Replaced the eventHub listeners and now deleting release actions via strapi.hooks

Test:
- Add entries of content-type to release
- Disable draft and publish
- The entries of content-type should be removed from the release
- Add entries of content-type to release
- Delete content-typ
- The entries of content-type should be removed from the release

File: packages/core/content-releases/server/src/migrations/index.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    // if d&p was disabled remove unpublish content before sync
+    if (
+      contentTypesUtils.hasDraftAndPublish(oldContentType) &&
+      !contentTypesUtils.hasDraftAndPublish(contentType)
+    ) {
+      await strapi.db
+        ?.queryBuilder(RELEASE_ACTION_MODEL_UID)
+        .delete()
+        .where({ contentType: uid })
+        .execute();
```
[the remark this answers is not in the document]
Comment (markkaylor) [reply packages/core/content-releases/server/src/migrations/index.ts:37]: It was a recommendation from Alex but now I can't remember why exactly... :(

File: packages/core/content-releases/server/src/migrations/index.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+  for (const uid in contentTypes) {
+    if (!oldContentTypes[uid]) {
+      continue;
+    }
+
+    const oldContentType = oldContentTypes[uid];
+    const contentType = contentTypes[uid];
+
+    // if d&p was disabled remove unpublish content before sync
```
Comment (markkaylor) [inline packages/core/content-releases/server/src/migrations/index.ts:28]: I will remove this comment

**Review by Feranchz — APPROVED** [review]
I tried all these cases locally and everything is working great 🚀 excellent work here

**Pull request merged**